### PR TITLE
feat: Add missing 'md' size

### PR DESCRIPTION
**Pull request by FelipePaluco**
Signed-off-by: Felipe Paluco <fepaluco@gmail.com>

Closes carbon-design-system/carbon-components-angular#2293

#### Changelog

**New**

* Adds missing 'md' size for ibm-modal component

### CODE REVIEW

**Comment by FelipePaluco** [comment]
I have read the DCO document and I hereby sign the DCO.

**Comment by FelipePaluco** [comment]
recheck

File: src/modal/modal.component.ts [hunk 1]
```diff
@@ -141,7 +142,7 @@ export class Modal implements AfterViewInit, OnChanges, OnDestroy {
 	/**
 	 * Size of the modal to display.
 	 */
-	@Input() size: "xs" | "sm" | "lg";
+	@Input() size: "xs" | "sm" | "md" | "lg";
```
Comment (Akshat55) [inline src/modal/modal.component.ts:145]:
```suggestion
	@Input() size: "xs" | "sm" | "md" | "lg" = "md";
```

The default size of modal is md (`carbon-components` indirectly sets the size to be `md`).

File: src/modal/modal.stories.ts [hunk 2]
```diff
@@ -242,7 +242,7 @@ class AlertModalStory {
 
 const getOptions = (options = {}) => {
 	return Object.assign({}, {
-		size: select("size", [null, "xs", "sm", "lg"], null)
+		size: select("size", [null, "xs", "sm", "md", "lg"], null)
```
[the remark this answers is not in the document]
Comment (Akshat55) [reply src/modal/modal.stories.ts:245]: Could you also all of the sizes in this story file - I noticed there are some incorrect sizes like `default` that should be there. So we should only be allowing `"xs", "sm", "md", "lg"` sizes

File: src/modal/modal.component.ts (hunk 1 above)
Comment (FelipePaluco) [reply src/modal/modal.component.ts]: solved on [a28466b](https://github.com/carbon-design-system/carbon-components-angular/pull/2365/commits/a28466bd8b69484176b87c3d52374ed52d06a3ec)

File: src/modal/modal.stories.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (FelipePaluco) [reply src/modal/modal.stories.ts]: solved on [a28466b](https://github.com/carbon-design-system/carbon-components-angular/pull/2365/commits/a28466bd8b69484176b87c3d52374ed52d06a3ec)

**Review by Akshat55 — APPROVED** [review]

**Review by zvonimirfras — APPROVED** [review]

**Pull request merged**